packages/styled-components/src/sheet/HoistedTag.js

```diff
@@ -4,19 +4,22 @@ import type { HoistedTag, GroupedTag, Tag } from './types';
 import { DefaultGroupedTag } from './GroupedTag';
 import { WindowedTag } from './WindowedTag';
 
-const hoistedRuleRe = /\s*@(font-face|import)/;
+const hoistedRuleRe = /\s*@import/;
 
 export class DefaultHoistedTag implements HoistedTag {
   hoistedTag: GroupedTag;
 
   normalTag: GroupedTag;
 
-  length: number;
+  /*
+  The number of Groups in the tag
```

Comment:
small nit: Let’s keep these as single line comments please 😅 personally is turn off our empty line rule between property type declarations as well, but we haven’t found an opportunity to migrate some rules

Reply:
I actually just forgot to remove this comment after renaming it, so thanks for pointing it out 😅